packages/ui/public/stylus.html

```diff
@@ -0,0 +1,82 @@
+  <link rel="apple-touch-icon" sizes="144x144" href="/favicons/apple-icon-144x144.png" />
+  <link rel="apple-touch-icon" sizes="152x152" href="/favicons/apple-icon-152x152.png" />
+  <link rel="apple-touch-icon" sizes="180x180" href="/favicons/apple-icon-180x180.png" />
+  <link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png" />
+  <link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png" />
+  <link rel="icon" type="image/png" sizes="96x96" href="/favicons/favicon-96x96.png" />
+  <link rel="icon" type="image/png" sizes="192x192" href="/favicons/favicon-192x192.png" />
+  <meta name="msapplication-TileColor" content="#ffffff">
+  <meta name="msapplication-TileImage" content="/ms-icon-144x144.png">
+  <meta name="theme-color" content="#ffffff">
+
+  <!-- Scripts and CSS -->
+  <link rel="preload" href="/fonts/Inter-VariableFont_opsz,wght.ttf" as="font" type="font/woff2" crossorigin>
+  <link rel='stylesheet' href='/build/standalone.css'>
+  <script async src='/build/embed.js'></script>
+</head>
+<body>
+
+  <div class="header container flex flex-row justify-between items-center overflow-auto">
+    <div class="flex flex-row">
+      <a href="https://www.openzeppelin.com/" target="_blank" rel="noopener noreferrer"><img src="/logo.svg" alt="OpenZeppelin" class="logo" width="160"></a>
+    </div>
+    <div class="flex flex-row items-center">
+      <a class="link" href="https://forum.openzeppelin.com/" target="_blank" rel="noopener noreferrer">Forum</a>
+      <a class="link" href="https://docs.openzeppelin.com/" target="_blank" rel="noopener noreferrer">Docs</a>
+      <a class="link" href="https://github.com/OpenZeppelin/" target="_blank" rel="noopener noreferrer"><img src="/icons/github-mark.svg" width="20" alt="GitHub OpenZeppelin"></a>
+      <a class="link" href="https://x.com/OpenZeppelin" target="_blank" rel="noopener noreferrer"><img src="/icons/x-logo.svg" width="20" alt="Twitter/X"></a>
+    </div>
+  </div>
+  <div class="nav">
+    <a class="switch switch-solidity" href="/"><img src="/icons/solidity.svg" alt="solidity">Solidity</a>
+    <a class="switch switch-cairo" href="/cairo"><img src="/icons/cairo.svg" alt="cairo">Cairo</a>
+    <a class="switch switch-stellar" href="/stellar"><img src="/icons/stellar.svg" alt="stellar">Stellar</a>
+    <a class="switch switch-stylus active" href="#"><img src="/icons/stylus_active.svg" alt="stylus">Stylus</a>
+  </div>
+  <div class="wizard-container">
+
+    <oz-wizard class="wizard" data-sync-url="fragment" data-lang="stylus"></oz-wizard>
+  </div>
+
+  <footer>
+    <p>© <a href="https://openzeppelin.com" target="_blank" rel="noopener noreferrer">OpenZeppelin</a> 2017-2025 |&nbsp;<a href="https://openzeppelin.com/privacy" target="_blank" rel="noopener noreferrer">Privacy</a> |&nbsp;<a href="https://openzeppelin.com/tos" target="_blank" rel="noopener noreferrer">Terms of Service</a></p>
+  </footer>
+
```

Comment:
@ericglau what do you think about calculating the current copyright year automatically?
```suggestion
  <footer>
    <p>© <a href="https://openzeppelin.com" target="_blank" rel="noopener noreferrer">OpenZeppelin</a> 2017-<span id="currentYear"></span> |&nbsp;<a href="https://openzeppelin.com/privacy" target="_blank" rel="noopener noreferrer">Privacy</a> |&nbsp;<a href="https://openzeppelin.com/tos" target="_blank" rel="noopener noreferrer">Terms of Service</a></p>
  </footer>
```
and then we could add a script below:
```javascript
<script>
    document.getElementById('currentYear').textContent = new Date().getFullYear();
</script>
```

Reply:
Makes sense. A similar approach is used for the docs [here](https://github.com/OpenZeppelin/docs.openzeppelin.com/blob/master/ui/theme/partials/footer.hbs#L3) and calculated [here](https://github.com/OpenZeppelin/docs.openzeppelin.com/blob/master/ui/src/js/copyright-date.js).

Could you make the change for the other langs as well (index.html, cairo.html, stellar.html)?